### PR TITLE
Align printed symbols | omit "SC(0)" | etc

### PR DESCRIPTION
**Pull request by VelocityRa**

### CODE REVIEW

File: ccc/util.cpp [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
 
+#include <filesystem>
+#include <fstream>
+#include <iterator>
+#include <vector>
+#include <cstdint>
+
+std::vector<uint8_t> read_file_bin(const std::filesystem::path& path) {
+	std::basic_ifstream<uint8_t> file{ path, std::ios::binary };
+	return { std::istreambuf_iterator<uint8_t>{file}, {} };
```
Comment (chaoticgd) [inline ccc/util.cpp:11]: For some reason this is crashing for me:

    terminate called after throwing an instance of 'std::bad_cast'
      what():  std::bad_cast
    
    Program received signal SIGABRT, Aborted.
    __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
    50	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
    (gdb) bt
    #0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
    #1  0x00007ffff7bb6859 in __GI_abort () at abort.c:79
    #2  0x00007ffff7e3c951 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
    #3  0x00007ffff7e4847c in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
    #4  0x00007ffff7e484e7 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
    #5  0x00007ffff7e487ed in __cxa_rethrow () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
    #6  0x000055555557f78a in void std::vector<unsigned char, std::allocator<unsigned char> >::_M_range_initialize<std::istreambuf_iterator<unsigned char,     std::char_traits<unsigned char> > >(std::istreambuf_iterator<unsigned char, std::char_traits<unsigned char> >, std::istreambuf_iterator<unsigned char,     std::char_traits<unsigned char> >, std::input_iterator_tag) ()
    #7  0x000055555557f43f in std::vector<unsigned char, std::allocator<unsigned char> >::vector<std::istreambuf_iterator<unsigned char, std::char_traits<unsigned char>     >, void>(std::istreambuf_iterator<unsigned char, std::char_traits<unsigned char> >, std::istreambuf_iterator<unsigned char, std::char_traits<unsigned char> >,     std::allocator<unsigned char> const&) ()
    #8  0x000055555557eece in read_file_bin(std::filesystem::__cxx11::path const&) ()
    #9  0x000055555557687c in read_program_image(std::filesystem::__cxx11::path) ()
    #10 0x0000555555571cbc in main ()

I must admit I don't know much about the C++ stream iterators so I don't know why, maybe just try a different implementation.

File: ccc/elf.cpp [hunk 2]
```diff
@@ -1,22 +1,13 @@
...
-	fseek(file, whence_you_came, SEEK_SET);
-	return size;
-}
-
 ProgramImage read_program_image(fs::path path) {
-	FILE* file = fopen(path.c_str(), "rb");
-	verify(file, "error: Failed to open file.\n");
-	u64 size = size_in_bytes(file);
+	if (!fs::exists(path)) {
+		verify(false, "error: file doesn't exist");
```
Comment (chaoticgd) [inline ccc/elf.cpp:5]: You can use verify_not_reached here.

**Review by chaoticgd — CHANGES_REQUESTED** [review]

File: ccc/elf.cpp (hunk 2 above)
Comment (chaoticgd) [reply ccc/elf.cpp:5]: Or even better, just verify(fs::exists(), ...). Also remember the \n.

**Comment by VelocityRa** [comment]
Ready.

**Comment by VelocityRa** [comment]
Ok ready.

**Comment by VelocityRa** [comment]
Pushed another thing that might trip up some sane compilers (ie. not MSVC).

**Comment by VelocityRa** [comment]
Github fucked up, re-opened PR #2.

**Pull request closed without merging**